In [ ]:
# Q1) Given to us is the time required for service transmissions are Normally Distributed with 
# i) mean = 45 , ii) Standard deviation = 8
#First finding the Probability of service manager meeting the customers need.
#Let 'a' be the Probability of the service Manager meeting the Customer Needs.

from scipy import stats

a=stats.norm.cdf(50,loc=45,scale=8)
print('a =',a)
print('The Probability of serving manager meeting the customer needs = ',round(a*100,2),'%')

a = 0.7340144709512995
The Probability of serving manager meeting the customer needs =  73.4 %


In [ ]:
# Probabilty of not meeting the Customers need.
print('1-a = ',1-a)
print('The Probability of Service Manager not meeting the needs = ',round((1-a)*100,2),'%')

1-a =  0.26598552904870054
The Probability of Service Manager not meeting the needs =  26.6 %


Therefore, the Probability of the service manager not able to meet the needs of the customer is 26.6 %.Which is near to option (b) of 0.2676.

Hence , we conclude that the answer for the question is option (b).

In [ ]:
# Q2) a) Given to us is current ages of 400 clerical employees in an insurance claim company are
# Normally distributed with mean = 38 and standard deviation = 6. 
# we are required to find whether:

# i) Employees older than age of 44.

from scipy import stats
a=stats.norm.cdf(44,loc=38,scale=6)
a

0.8413447460685429

In [ ]:
# To get the no.of employees out of 400,we multiply it with 400.
print('No.of Employees older than age of 44 are ',round(a*400))

No.of Employees older than age of 44 are  337


There are 337 Employees older than age of 44.

In [ ]:
# 2) a) ii)Or Employees of the age between 38 and 44.
b=stats.norm.cdf(44,loc=38,scale=6)-stats.norm.cdf(38,loc=38,scale=6)
print(round(b*100,3),'%')

34.134 %


In [ ]:
print('No of Employess of the age between 38 and 44 are',round(b*400))

No of Employess of the age between 38 and 44 are 137


 There are 137 Employees working under between the age of 38 and 44.

By comparing the results from the above statements,we can say that the No.of Employees working are of the age older than 44 are more than that of the Employees working under the age of 38 and 44.

Hence, we say that the stament is True.

In [ ]:
# 2 b) A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.

# Percentage of Employees under the age of 30
c=stats.norm.cdf(30,loc=38,scale=6)
print(round(c*100,2),'%')

9.12 %


Therefore, we conclude that the Percentage of Employees under the age of 30 is 
9.12%.

In [ ]:
# To get No.of Employees under the age of 30,we multiply with 400.
print(round(c*400))

36


Therefore , we conclude that there are '36' Employees under the age of 30.

In [ ]:
# Q4)
# Given to us is mean = 100 and standard deviation = 20.
# ->  We need to find two values between which the Probability is 99%.
# -> By going through options and finding which gives the probability of 99%.

# i) a = 90.5 and b = 105.9

from scipy import stats

a = stats.norm.cdf(105.9,loc=100,scale=20) - stats.norm.cdf(90.5,loc=100,scale=20)
print(round(a*100),'%')

30 %


In [ ]:
# ii) a=80.2  and b = 119.8
a = stats.norm.cdf(119.8,loc=100,scale=20) - stats.norm.cdf(80.2,loc=100,scale=20)
print(round(a*100),'%')

68 %


In [ ]:
# iii) a = 22 and b = 78
a = stats.norm.cdf(78,loc=100,scale=20) - stats.norm.cdf(22,loc=100,scale=20)
print(round(a*100),'%')

14 %


In [ ]:
# - >  iv) a = 48.5 and b = 151.5
a = stats.norm.cdf(151.5,loc=100,scale=20) - stats.norm.cdf(48.5,loc=100,scale=20)
print(round(a*100),'%')

99 %


In [ ]:
# v) a = 90.1 and  b = 109.9
a = stats.norm.cdf(109.9,loc=100,scale=20) - stats.norm.cdf(90.1,loc=100,scale=20)
print(round(a*100),'%')

38 %


From the above options, we find that the 
-> (iv) option , that is, when a = 48.5 and b = 151.5 are the two values ,
 symmetric about the mean , such that the probability of the random variable taking a value between them is 0.99.